src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -239,11 +317,45 @@ public void setCoastMode() {
     rightLeadMotor.setNeutralMode(NeutralMode.Coast);
   }
 
+  public RamseteCommand getRamseteCommand(Trajectory trajectory) {
+    System.out.println(this);
```

Comment:
Probably want to remove this line. prints can slow down processing. Not sure how often this is called so not sure if this is a huge issue or not.

Reply:
This is only called when robot first powers on, but it is debug so I'll remove it